apps/full-site-editing/full-site-editing-plugin/full-site-editing/utils/class-wp-template.php

```diff
@@ -1,14 +1,16 @@
 <?php
 /**
- * A8C WP Template file.
+ * WP Template file.
  *
- * @package full-site-editing
+ * @package A8C\FSE
  */
 
+namespace A8C\FSE;
+
 /**
- * Class A8C_WP_Template
+ * Class WP_Template
  */
-class A8C_WP_Template {
+class WP_Template {
```

Comment:
Just noting that we'll likely need to update the usage here https://github.com/Automattic/themes/blob/master/modern-business/header.php#L28 and similar one in `footer.php`.

Reply:
@vindl do you prefer keeping this as A8C_WP_Template?

Reply:
No, I prefer the namespaced one. Updated in https://github.com/Automattic/themes/pull/1174/commits/811a2cab9b5665597ec80d3874fa0a2d78414ca8.